intfsyncd/intfsync.cpp

```diff
@@ -47,6 +47,11 @@ void IntfSync::onMsg(int nlmsg_type, struct nl_object *obj)
         return;
 
     key = LinkCache::getInstance().ifindexToName(rtnl_addr_get_ifindex(addr));
+
+    /* Don't sync lo, eth0, and docker0 routes */
+    if (key == "lo" || key == "eth0" || key == "docker0")
+        return;
```

Comment:
Then we need to get the list of ports, lags and vlans during the runtime. For now hard coded values can solve the issue but later we might have to discuss about what approach is better.